test/recipes/90-test_buffers.t

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env perl
+# Copyright 2015-2017 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+
+setup("test_buffers");
+
+plan skip_all => "No TLS/SSL protocols are supported by this OpenSSL build"
+    if alldisabled(grep { $_ ne "ssl3" } available_protocols("tls"));
```

Comment:
"No suitable..." sounds find to me. But, in this case, I will be removing the SSLv3 restriction.